.vscode/cspell.json

```diff
@@ -134,8 +147,12 @@
     "guids",
     "hanaonazure",
     "hdinsight",
+    "hexlify",
```

Comment:
No action here. Just wanted to say out loud: Strange... One would think that python standard libraries would be in the cspell python dictionary. This one is not.